hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -1454,7 +1450,7 @@ class ShrinkPass:
     shrinks = attr.ib(default=0)
     deletions = attr.ib(default=0)
 
-    def step(self):
+    def step(self, random=False):
```

Comment:
Minor: Since we have so many `random` arguments that take `Random` instances, and this isn't one of them, I like renaming this to something more like `random_order`.